.travis.yml

```diff
@@ -1,14 +1,15 @@
 language: rust
 sudo: false
 rust:
```

Comment:
Hm which part requires 1.4.0? Ideally this would continue to compile on 1.0.0 as long as possible, but if it's too hard it may not be worth it